types/index.d.ts

```diff
@@ -1,4 +1,7 @@
-declare module '*.svg' {
```

Comment:
move this file to `src/env.d.ts` and then insert `/// <reference types="vite/client" />` to the first line.

See also: https://vitejs.dev/guide/env-and-mode.html#env-files

Also I think `declare module "*.svg" { ... }` can be removed by adding `/// <reference types="vite-plugin-svgr/client" />`

See https://github.com/pd4d10/vite-plugin-svgr/blob/main/client.d.ts and you will see SVG module is already defined

Reply:
I had both of those references inside the vite.config.ts file, since in other repos we have it setup like that. But it seemed, at least for the svgr one, to not work from there. When I move them into the env file, it works.

Reply:
No need to add it to `vite.config.ts`. It is not ".d.ts". TypeScript does not use vite.config.ts to load type definitions. It should be `env.d.ts`.